cita-executor/core/src/libexecutor/executor.rs

```diff
@@ -66,28 +68,24 @@ impl Executor {
         // TODO: Can remove NUM_COLUMNS(useless)
         let config = Config::with_category_num(NUM_COLUMNS);
         let rocks_db = RocksDB::open(&data_path, &config).unwrap();
-        let trie_db = Arc::new(TrieDB::new(Arc::new(rocks_db)));
+        let db = Arc::new(rocks_db);
+        let trie_db = Arc::new(TrieDB::new(db.clone()));
```

Comment:
`state_db` might be better than `trie_db`, please check:

```rs
 pub fn lazy_execute(
        &mut self,	        &mut self,
        // state_db: &StateDB,	        state_db: Arc<CitaTrieDB>,
}
```

Reply:
Change the value-name trie_db to state_db.